docs/CHANGELOG.asciidoc

```diff
@@ -28,6 +28,360 @@ This section summarizes the changes in each release.
+* Registers `savedObjectMetaData` so Maps saved objects show up in dashboard add panel {pull}32895[#32895]
+* Populates WMS layers from `getCapabilities` response {pull}32342[#32342]
+* Splits settings into layer and source panels {pull}33788[#33788]
+* Adds layer-specific filtering {pull}33209[#33209]
+* Adds clone layer to layer actions {pull}33853[#33853]
+* Automatically disables filter by bounds for indexes with small doc counts {pull}34456[#34456]
+* Adds ability to expand and collapse Layer TOC {pull}34506[#34506]
+* Ignores global query layer setting {pull}35542[#35542]
+* Shows dynamic style ranges in legend {pull}35417[#35417]
+* Shows icon when layer is not visible in TOC {pull}36250[#36250]
+* Shows minusInCircle icon when layer has no data {pull}36457[#36457]
+* Only shows vector style properties used by source in editor {pull}36280[#36280]
+* Supports multiple basemap styles {pull}36074[#36074]
+* Locks tooltip in place with click {pull}32733[#32733]
+* Uses dual range component {pull}32273[#32273]
+* Adds filter actions to tooltips {pull}33635[#33635]
+* Adds draw filter action to layer {pull}33686[#33686]
+* Adds WKT and geohash string support {pull}34472[#34472]
+
+Kibana App::
+* Allows multiple values for percentile ranks in Time Series Visual Builder {pull}33642[#33642]
+* Allows non numeric fields in Top Hits in Time Series Visual Builder {pull}35661[#35661]
+* Adds Include and Exclude fields to the Terms group in Time Series Visual Builder {pull}34153[#34153]
+* Adds ability to submit Save modals using the Enter key {pull}34417[#34417]
+* Improves KQL error messages {pull}34900[#34900]
+* Uses embeddable registry in add panel {pull}31400[#31400]
+* Shows a reload dialog when users change a setting requiring a page reload {pull}34780[#34780]
+* Improves scripted field script editor {pull}36064[#36064]
+* Changes default quick ranges {pull}35066[#35066]
+
+Kibana Home &amp; Add Data::
+* Injects anomalies into the Kibana sample datasets to enable ML sample jobs {pull}36714[#36714]
+* Allows plugins to register additional view data links in sample data {pull}33052[#33052]
+
+Logs::
+* Adds customizable columns {pull}34916[#34916]
+* Adds column headers {pull}36467[#36467]
+* Adds flyout action menu with Uptime link {pull}36721[#36721]
+
+Machine Learning::
+* Adds Kibana sample dataset data recognizer modules  {pull}35138[#35138]
+* Adds File data viz filter to Index Management link {pull}36166[#36166]
+* Adds query bar to Anomaly Explorer for filtering of anomaly results for one or more influencers {pull}31763[#31763]
+* Adds the ability to add and remove filters in the Anomaly Explorer via the Anomalies table  {pull}33377[#33377]
+* Adds the ability to add and remove filters in the Anomaly Explorer via the Top influencers list {pull}33626[#33626]
+* Converts Job Selector to React {pull}35638[#35638]
+* Always show time warning message in delete modal in Job Management {pull}36729[#36729]
```

Comment:
For the last part, how about:

`the delete modal for Job Management`

The double `in`s are causing pause.